src/FunctionApps/python/GenerateCSVs/elr.py

```diff
@@ -1,8 +1,39 @@
 from typing import List, Any
-from GenerateCSVs.patient import PATIENT_COLUMNS
+from GenerateCSVs.patient import PATIENT_COLUMNS, parse_patient_resource
 
-ELR_COLUMNS = PATIENT_COLUMNS + []
+
+ELR_COLUMNS = PATIENT_COLUMNS + ["loinc_code", "result", "effective_date"]
+
+
+def extract_covid_test(observation: dict) -> dict:
+    """
+    Given an observation FHIR profile, determine whether the profile
+    contains information related to a COVID test or COVID status.
+    If yes, collect the relevant lab code, test result, and test
+    date into a list.
+    """
+    print(observation)
+    code = observation["code"]["coding"][0]
```

Comment:
I know I'm just being paranoid since code.coding is a required field, but should we be doing more checking to ensure good data?  Right now, if either code or code.coding isn't valued (or malformed), I think it will throw an error that will terminate the whole processing batch.  Same general comment applies in other areas as well.

Reply:
We decided that for right now, we don't need to be that defensive. You're right that for a prod-like long-term solution we would want to use `try/except` here and be more defensive, but these CSVs are a one-time, one-off solution that doesn't represent any sort of long-term legitimate attempt. Our philosophy has largely been to be defensive later as the data requires us rather than get too complicated right now.

Reply:
:+1: sounds good to me.